enterprise_subsidy/apps/transaction/tests/test_signal_handlers.py

```diff
@@ -206,6 +214,8 @@ def test_handle_lc_enrollment_revoked(
             assert any(re.search(expected_log_regex, log) for log in logs.output)
         if expected_reverse_transaction_called:
             mock_reverse_transaction.assert_called_once_with(transaction, unenroll_time=enrollment_unenrolled_at)
+        if expected_cancel_transaction_external_fulfillment_called:
+            mock_cancel_transaction_external_fulfillment.assert_called_once_with(transaction)
```

Comment:
and here's where we now assert that `cancel_transaction_external_fulfillment()` actually gets called even if the transaction is not refundable.